dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:latest
+RUN apt-get update && apt-get upgrade -y
+RUN apt-get install python3-pip python3 python-is-python3 -y
```

Comment:
Pin to Python 3.9

Reply:
Pinning 3.8 is great. Kind of wild to have to select a timezone, must be a way around that 🤔

Reply:
it has to do with tzdata which i think is a requirement in apt-get in python3.9

Reply:
```ARG DEBIAN_FRONTEND=noninteractive
ENV TZ=Europe/Moscow
RUN apt-get install -y tzdata```

 if we want to pin to 3.9

https://serverfault.com/questions/949991/how-to-install-tzdata-on-a-ubuntu-docker-image